### ISSUE TITLE
[Bug] Offset on keypoint coordinates in output 

### ISSUE THREAD

**Issue by LjIA26** [open-mmlab/mmpose#2629]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmpose/issues) and [Discussions](https://github.com/open-mmlab/mmpose/discussions) but cannot get the expected help.
- [X] The bug has not been fixed in the latest version(https://github.com/open-mmlab/mmpose).

### Bug

Thank you for your work. I found a bug. 

There is an offset of the printed coordinates in the json file. We can only get the true coordinates when we set the radius of the circle that gets printed as 0. Otherwise, the printed coordinates are wrong and it prints the outer right point on the diameter of the circle that gets plotted. This should get fixed. 

### Reproduces the problem - code sample

Not a problem but a bug

**Comment by Ben-Louis** [comment]
Thanks for your feedback. Could you please give an example to illustrate this problem?

**Comment by LjIA26** [comment]
Unfortunately, I can't share the images, but I can share the keypoints that got printed. 

Because I used the keypoints on the prediction folder and then I plotted them myself on top of the images, that's how I noticed it.  

with radius equal to zero: 

```
      [
        366.69140625,
        343.91015625
      ],
      [
        349.01953125,
        252.60546875
      ],
      [
        343.12890625,
        190.75390625
      ],

```
with radius equal to 1: 

```
      [
        367.69140625,
        342.91015625
      ],
      [
        350.01953125,
        251.60546875
      ],
      [
        344.12890625,
        189.75390625
      ],


```

**Comment by Ben-Louis** [comment]
Thank you for bringing this issue to our attention. This bug should have been fixed in #2631. We appreciate any additional feedback you may have on your user experience.
